helper/validate_password.go

```diff
@@ -0,0 +1,78 @@
+package helper
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/muesli/crunchy"
+)
+
+const (
+	minPasswordLengths = 14
+)
+
+var (
+	ErrLength = errors.New("password is too short")
```

Comment:
we need to notify user about min length e.g. "password is too short" --> "Please enter at least 14 characters"